scripts/docker/CC-Devel/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM ubuntu:16.04
+
+RUN \
+    apt-get update \
+    && apt-get install --yes \
```

Comment:
Please format this better so that the arguments of `install` are indented to be visibly under `install` command, like this:

```Dockerfile
RUN apt-get update \
  && apt-get install --yes \
    openjdk-8-jdk-headless \
    libboost-...
```